core/src/test/java/com/ibm/wala/core/tests/callGraph/ClassConstantTest.java

```diff
@@ -79,4 +79,14 @@ public void testClassConstants()
     // make sure call to hashCode from main
     assertTrue(cg.hasEdge(mainMethodNode, hashCodeNodes.iterator().next()));
   }
+
+  @Test
+  public void classHierarchyToJson() throws ClassHierarchyException, IOException {
+    AnalysisScope scope =
+        CallGraphTestUtil.makeJ2SEAnalysisScope(
+            TestConstants.WALA_TESTDATA, CallGraphTestUtil.REGRESSION_EXCLUSIONS);
+    ClassHierarchy cha = ClassHierarchyFactory.make(scope);
+    Object unused = cha.toJson();
+    // assertEquals("", json);
```

Comment:
Don't we need to write some actual assertions about what we expect to be in the final JSON string?